stock_request/models/stock_request.py

```diff
@@ -283,6 +280,7 @@ def _prepare_procurement_values(self, group_id=False):
         """
         return {
             "date_planned": self.expected_date,
+            "date_deadline": self.expected_date,
```

Comment:
I would prefer send date instead date time but it really depends on business here :thinking:

Reply:
In fact that this is an other matter that should be address from an other module, this proposed fix is similar from what's  done in sale_stock module while selling goods, date_deadline  is set from date_order which is a datetime

Reply:
done by https://github.com/OCA/stock-logistics-workflow/pull/1226